src/binary/logger.rs

```diff
@@ -39,8 +75,7 @@ impl log::Log for LockedLogger {
 
     fn log(&self, record: &log::Record) {
         let mut logger = self.0.lock();
-        writeln!(logger, "{}:    {}", record.level(), record.args()).unwrap();
-        logger.add_vspace(LOG_SPACING);
```

Comment:
Is there any specific reason for removing this? This gave us a bit of extra padding around individual log messages (which might be multi-line).

Reply:
it still exists, but it is now located here: https://github.com/rust-osdev/bootloader/pull/267/files#diff-516264e516b0fbbd881351d365dbf0af59b2a151f200ce3fdc8abd4f0b99cbf7R106

I think, it makes more sense there.

Reply:
There is a slight difference for multi-line messages. Before, the padding was only added between "paragraphs", now it is also added for every line break in long messages. But either way is fine with me, so we can keep it like this.